Sources/Shout/SFTP.swift

```diff
@@ -110,6 +110,32 @@ public class SFTP {
     ///   - permissions: the file permissions to create the new file with; defaults to FilePermissions.default
     /// - Throws: SSHError if local file can't be read or upload fails
     public func upload(localURL: URL, remotePath: String, permissions: FilePermissions = .default) throws {
+        let data = try Data(contentsOf: localURL, options: .alwaysMapped)
+        try upload(data: data, remotePath: remotePath, permissions: permissions)
+    }
+    
+    /// Upload a file from the local device to the remote server
```

Comment:
I think this should read "Upload a string to a file on the remote server" or something like that